manager/eris-mint/state/state_test.go

```diff
@@ -463,7 +464,7 @@ func TestCreates(t *testing.T) {
 			PubKey:   acc0PubKey,
 		},
 		Address:  acc1.Address,
-		GasLimit: 10000,
+		GasLimit: 100000,
```

Comment:
There just wasn't enough gas for the hardcoded bytecode that is used by this test